test/e2e/healthcheck_run_test.go

```diff
@@ -261,6 +261,16 @@ var _ = Describe("Podman healthcheck run", func() {
 		Expect(ps).Should(ExitCleanly())
 		Expect(ps.OutputToStringArray()).To(HaveLen(2))
 		Expect(ps.OutputToString()).To(ContainSubstring("hc"))
+
+		// Test hc logs to not include exec-died events
+		hcLogs := podmanTest.Podman([]string{"events", "--stream=false", "--filter", "event=exec_died", "--filter", "event=exec", "--since", "1m"})
+		hcLogs.WaitWithDefaultTimeout()
+		Expect(hcLogs).Should(ExitCleanly())
+		hcLogsArray := hcLogs.OutputToStringArray()
+		Expect(hcLogsArray).To(HaveLen(2))
+		Expect(hcLogsArray[0]).To(ContainSubstring("exec"))
+		Expect(hcLogsArray[1]).To(ContainSubstring("exec_died"))
```

Comment:
you should add a comment and make clear that we do not expect any exec events from healthchecks, we only have these two because we run one podman exec command above

Reply:
done